src/qt/walletmodel.cpp

```diff
@@ -120,12 +119,18 @@ void WalletModel::pollBalanceChanged()
 
 void WalletModel::checkBalanceChanged(const interfaces::WalletBalances& new_balances)
 {
-    if(new_balances.balanceChanged(m_cached_balances)) {
+    const auto& cached_balace = getCachedBalance();
```

Comment:
In 0da631685626c84934735cf9ddef5cca834180a2 "GUI: add getter for WalletModel::m_cached_balances field."

nit: typo

```suggestion
    const auto& cached_balance = getCachedBalance();
```